docs/mimxrt/quickref.rst

```diff
@@ -301,13 +301,16 @@ There are up to four hardware SPI channels that allow faster transmission
 rates (up to 30Mhz).  Hardware SPI is accessed via the
 :ref:`machine.SPI <machine.SPI>` class and has the same methods as software SPI above::
 
-    from machine import SPI
+    from machine import SPI, Pin
 
-    spi = SPI(0, 10000000)
+    cs_pin = Pin(6, Pin.OUT, value=1)
```

Comment:
I think you meant to keep the `spi = ...` line as well

Reply:
Thank you for reviewing. Of course that line is needed.

Reply:
And it shows that I was not sure about the default for the cs option.